packages/components/package.json

```diff
@@ -94,6 +94,8 @@
     "ember-cli-string-helpers": "^6.1.0",
     "ember-cli-terser": "^4.0.2",
     "ember-concurrency": "^3.1.1",
+    "ember-copy": "^2.0.1",
```

Comment:
@alex-ju once we will separate the showcase to its own standalone app, this medley of packages for the components vs the showcase should be fixed once for all

Reply:
In the end I've removed the dependency and used a simple "clone" function.